application/controller/index.go

```diff
@@ -0,0 +1,170 @@
+package controller
+
+import "github.com/google/wire"
+
+var Set = wire.NewSet(NewMusicController)
+
+type HTTPCode int32
+
+// https://github.com/googleapis/googleapis/blob/643311525df18aff3090d13449db4e9c2e9df419/google/rpc/code.proto#L32
+const (
+	// Not an error; returned on success
+	//
+	// HTTP Mapping: 200 OK
+	Ok HTTPCode = iota
```

Comment:
Is the use of this const some kind of enum?

as i understand
```go
const Ok = 0
const Cancelled = 1
const Unknown = 2
...
```

but shouldn't the value be like below?
```go
const Ok = 200
const Cancelled = 499
const Unknown = 500
...
```

Reply:
Why don't you use this? [net/http#status](https://cs.opensource.google/go/go/+/refs/tags/go1.18.5:src/net/http/status.go;l=9)